specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2019-12-01-preview/apimnamedvalues.json

```diff
@@ -34,19 +34,19 @@
     }
   },
   "paths": {
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/properties": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/namedValues": {
       "get": {
         "tags": [
-          "Property"
+          "NamedValue"
         ],
-        "operationId": "Property_ListByService",
-        "description": "Lists a collection of properties defined within a service instance.",
+        "operationId": "NamedValue_ListByService",
+        "description": "Lists a collection of NamedValues defined within a service instance.",
         "externalDocs": {
           "url": "https://docs.microsoft.com/en-us/azure/api-management/api-management-howto-properties"
```

Comment:
>properties [](start = 93, length = 10)

we should ask the PM to rename the doc name :)

Reply:
I asked Mike to do this